src/Core/System/Capabilities.js

```diff
@@ -38,22 +38,7 @@ Capabilities.prototype.ioFile = function ioFile() {
 };
 
 Capabilities.prototype.isInternetExplorer = function isInternetExplorer() {
-    return /* @cc_on!@*/ false || !!document.documentMode;
+    return false || !!document.documentMode;
```

Comment:
The `/* @cc_on!*/` is a trick to detect IE < 11 - so it may be needed.
(but as IE < 11 isn't supported by three.js maybe this check is useless anyway)